minetest.conf.example

```diff
@@ -108,8 +108,9 @@
 #liquid_queue_purge_time = 0
 #    Liquid update interval in seconds
 #liquid_update = 1.0
-#    Enable transparent leaf textures, disable for speed
-#new_style_leaves = true
+#    Leaves style: fancy - all faces visible, simple - only outer faces,
+#    opaque - disable transparency
```

Comment:
my first look at this is a little confusing, maybe indent it or split it up like https://github.com/RealBadAngel/minetest/blob/d7a2523df82b2452726b4ef51d3b8ff7ca558636/minetest.conf.example#L159